README.md

```diff
@@ -16,27 +16,27 @@ pipenv install --dev
 Extract translatable text from an eQ survey with
 
 ```
-pipenv run ./scripts/run_translation_extract.sh <json_file> [output_directory]
+pipenv run python -m app.template_extractor <schema_file> <output_directory> <pot/csv>
```

Comment:
I think we would benefit from making using of a `Makefile` that runs these commands?

So we can just do:
``` sh
make translate SCHEMA=<schema_file> OUT_DIR=<output_directory> FORMAT=<pot/csv>
```

Thoughts?

Reply:
Given it's documented in the readme - I'd prefer people just followed that rather than we have another file/command to maintain. 

Part of the reason that all the bash scripts have been removed is they offer nothing over and above the python yet we still have to maintain them.